lib/generators/annotate/templates/auto_annotate_models.rake

```diff
@@ -13,6 +13,16 @@ if Rails.env.development?
       'position_in_fixture'     => 'before',
       'position_in_factory'     => 'before',
       'position_in_serializer'  => 'before',
+      'include_model'           => true,
```

Comment:
Since we're introducing the "types" option, why don't we use that here as well instead of having these?  Something similar to what you did above:

'types' => 'model,test,fixture,factory,serializer,scaffold,controller,helper'
or
'types' => %w(model test fixture factory serializer scaffold controller helper)

I need to figure out what the previous defaults since the new defaults seem to surprise quite a number of people (I can do that after this PR if necessary).

I think you're on the right track but I'll need to check this out on my machine to play with it a bit.